accord-core/src/main/java/accord/coordinate/CoordinateSyncPoint.java

```diff
@@ -128,7 +128,7 @@ void onPreAccepted(Topologies topologies, Timestamp executeAt, List<PreAcceptOk>
             topologies = node.topology().forEpoch(route, txnId.epoch());
             // TODO (required): consider the required semantics of a SyncPoint
             if (tracker.hasFastPathAccepted() && txnId.kind() == Kind.SyncPoint)
-                execute(adapter, node, topologies, route, FAST, txnId, txn, executeAt, deps, this);
+                execute(adapter, node, topologies, route, FAST, txnId, txn, txnId, deps, this);
```

Comment:
I don't think this is correct - sync points take an actual executeAt. If we intend to coordinate them like ExclusiveSyncPoint then we need to modify them to execute the same way (i.e. they need to update their kind to signify `awaitsOnlyDeps`, and need to use the same execution path as `ExclusiveSyncPoint` and `EphemeralRead`)

Reply:
This is the fast path, when we talked about this you said that fast path pre-accept always uses the transaction ID, not executeAt. This is how the fast path works for regular transactions. Why would inclusive sync points need to function differently?